waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim

```diff
@@ -784,9 +787,21 @@ proc mountRlnRelayDynamic*(node: WakuNode,
     rlnIndex = regIndexRes.value
     debug "peer is successfully registered into the membership contract"
   else:
+    info "Peer is already registered to the membership contract"
     keyPair = memKeyPair.get()
     rlnIndex = memIndex.get()
 
+  #Write KeyPair
+  let kp = %keyPair
+  writeFile("keyPair.txt", pretty(kp))
```

Comment:
Could the filenames be a `const` elsewhere which can be reused when reading/writing?

Reply:
Suggestion incorporated.